api/app/src/app.ts

```diff
@@ -10,8 +10,8 @@ import mountRoutes from "./routes/index";
 
 const app: Application = express();
 app.use(helmet()); // needs to come before any route declaration, including cors()
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.json({ limit: 2_000 }));
```

Comment:
As long as we're sure this is kb

Reply:
No, my bad, this param is in bytes, so should be 2_000_000... I'll just use "2mb"